From 0fc4fc87a490af99bb9688db133ead51e2b90db6 Mon Sep 17 00:00:00 2001
From: Hiroshi Miura <miurahr@linux.com>
Date: Fri, 30 Mar 2018 14:30:28 +0900
Subject: [PATCH 2/2] Fix win32 compilation

- rl2wms.c: strcasecmp detection and usage
- rl2import.c: not include unistd.h when win32

Signed-off-by: Hiroshi Miura <miurahr@linux.com>
---
 src/rl2import.c | 2 +-
 src/rl2wms.c    | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/rl2import.c b/src/rl2import.c
index 7694e7b..ea80559 100644
--- a/src/rl2import.c
+++ b/src/rl2import.c
@@ -41,7 +41,6 @@ the terms of any one of the MPL, the GPL or the LGPL.
  
 */
 
-#include <unistd.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
@@ -66,6 +65,7 @@ the terms of any one of the MPL, the GPL or the LGPL.
 #include <windows.h>
 #include <process.h>
 #else
+#include <unistd.h>
 #include <pthread.h>
 #endif
 
diff --git a/src/rl2wms.c b/src/rl2wms.c
index fafa6e6..c742ecf 100644
--- a/src/rl2wms.c
+++ b/src/rl2wms.c
@@ -50,6 +50,7 @@ the terms of any one of the MPL, the GPL or the LGPL.
 #include <curl/curl.h>
 #include <libxml/parser.h>
 
+#include "config.h"
 #include "rasterlite2/sqlite.h"
 
 #include "rasterlite2/rasterlite2.h"
-- 
2.16.2

